src/index.js

```diff
@@ -34,7 +34,7 @@ export default class EHoldings extends Component {
     router: PropTypes.object
   };
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() { // eslint-disable-line camelcase
```

Comment:
This is probably not the right approach for this case but I was not able to move these lines into a `componentDidUpdate` or similar side-effect allowable method. Any other suggestions welcome.

Reply:
If `this.context` is available in the constructor, it might work there

Reply:
No luck with that. Does not seem to be available in the constructor.
> Uncaught TypeError: Cannot read property 'addReducer' of undefined

Reply:
Oh hold on, I forgot that context gets passed as the second parameter of the `constructor`